modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase

```diff
@@ -35,6 +35,30 @@ definition {
 		}
 	}
 
+	@description = "LPS-176607 - Verify that there is a 'cancel' button being showed"
+	@priority = 2
+	test CanCancelOrganizationEdition {
+		task ("Given a user is on the Organization page") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "employee@liferay.com",
+				userLoginFullName = "Employee User");
+
+			EVPUtils.gotoEVP();
+
+			EVPNavigator.goToNavItem(navItem = "Organization");
+		}
+
+		task ("When a user edits an Organization") {
```

Comment:
Due to the description going to be changed, I think we could change the task to:` "When a user goes to organization edition page"`